apps/desktop/src/components/PasswordInput.tsx

```diff
@@ -30,12 +33,31 @@ export const PasswordInput = <T extends FieldValues, U extends Path<T>>({
   label = "Password",
   placeholder = "Enter your password",
   required = "Password is required",
+  checkPasswordStrength = false,
   minLength = MIN_LENGTH,
   validate,
   ...rest
 }: PasswordInputProps<T, U>) => {
   const { register } = useFormContext<T>();
   const [showPassword, setShowPassword] = useState<boolean>(false);
+  const { validatePassword, PasswordStrengthBar } = usePasswordValidation({
```

Comment:
nit: maybe `validatePasswordStrength` to avoid confusion with validate function from the input